lib/crash_analytics/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  firebase_crashlytics: ^2.0.7
+  firebase_crashlytics: ^2.5.1
 
```

Comment:
I guess this is not compatible with our Flutter version?

Reply:
Yeah, it's looks like that. I needed to upgrade `firebase_crashlytics`, because they fixed in `2.5.1` a macOS bug. Should I open a new PR where we upgrade to the latest Flutter version (if possible)?